index.html

```diff
@@ -60,7 +63,16 @@
       </div>
     </div>
   </div>
-    
+  
+  <div>
+
+  <div >
+    <a href="instruction.html"><img src="images/surprise.jpeg" class="surprise" style="width:100px;height:100px;"></a>
+  </div>
```

Comment:
If we make this a link, then the app needs to reload every time you go back from the instructions page. 

Instead, we can make the instruction menu a div, that is hidden by default (use CSS styling display: none;) and we can make this button show the instruction screen by setting its display to something other than none. 
Then, to hide it, use the back arrow on the instruction screen to set the display to none again.